tests/examples/run/test_classification.sh

```diff
@@ -8,3 +8,8 @@ python ${KAOLIN_HOME}/examples/Classification/pointcloud_classification.py \
     --modelnet-root /data/ModelNet10/ \
     --categories desk dresser \
     --epochs 1 --transforms-device cpu
+
+python ${KAOLIN_HOME}/examples/Classification/mesh_classification.py \
```

Comment:
In tests the CI catch the returned value of the script, if you run those two commands like that if the first command fails the return value is gonna be overriden by this one.

I recommend using command1 && command2